### PR TITLE
STY: Remove commented-out code (ruff:ERA001)

### PR DESCRIPTION
**Pull request by MartinThoma**

### CODE REVIEW

File: pypdf/_writer.py [hunk 1]
```diff
@@ -1832,6 +1830,8 @@ def remove_images(self, ignore_byte_string_object: bool = False) -> None:
         )
         for page in pages:
             page_ref = cast(DictionaryObject, self.get_object(page))
+            if not "/Contents" in page_ref:
+                return
```
Comment (MartinThoma) [inline pypdf/_writer.py:1834]: ROB: This change might be surprising in this PR

File: pypdf/_writer.py [hunk 2]
```diff
@@ -1832,6 +1830,8 @@ def remove_images(self, ignore_byte_string_object: bool = False) -> None:
         )
         for page in pages:
             page_ref = cast(DictionaryObject, self.get_object(page))
+            if "/Contents" not in page_ref:
+                return
```
Comment (MartinThoma) [inline pypdf/_writer.py:1834]: ROB: This change might be surprising in this PR

**Comment by MartinThoma** [comment]
@pubpub-zz Is this OK for you?

You were the most active developer on pypdf in the past year and I want to avoid removing helpful reminders. But I don't want to have dead code in the library :smile:

File: pypdf/_writer.py (hunk 2 above)
Comment (pubpub-zz) [reply pypdf/_writer.py:1834]: agree : the test reactivation below and this change would be better in another PR

**Review by pubpub-zz — COMMENTED** [review]
sounds good to me, those subject are globally closed
else agree with @MartinThoma 's comment

File: pypdf/_writer.py (hunk 2 above)
Comment (MartinThoma) [reply pypdf/_writer.py:1834]: You're right :+1: 

I moved this to https://github.com/py-pdf/pypdf/pull/1598

**Pull request merged**